packages/js/src/workouts/components/SocialInputPersonSection.js

```diff
@@ -39,37 +39,34 @@ function SocialInputPersonSection( { personId } ) {
 					</Alert>
 					<p>
 						{
-							createInterpolateElement(
+							addLinkToString(
 								sprintf(
+									// translators: %1$s and %2$s are replaced by opening and closing anchor tags
 									__(
-										// translators: %1$s and %2$s are replaced by opening and closing anchor tags
 										"In this step, you need to add the personal social profiles of the person your site represents. To do that, you should go to the %1$sUsers%2$s > Profile page in a new browser tab. Alternatively, ask the user or an admin to do it if you are not allowed.",
 										"wordpress-seo"
 									),
 									"<a>",
 									"</a>"
 								),
-								{
-									b: <b />,
-									// eslint-disable-next-line jsx-a11y/anchor-has-content
-									a: <a id="yoast-configuration-workout-user-page-link-1" href={ window.wpseoWorkoutsData.usersPageUrl } target="_blank" rel="noopener noreferrer" />,
-								}
+								window.wpseoFirstTimeConfigurationData.usersPageUrl,
+								"yoast-configuration-workout-user-page-link-1"
```

Comment:
I was wondering how this would work, but we don't use this component any longer after #18455 has been merged, right?

Reply:
Right, this is an old component and is removed in #18455!